File: example/javalib/testing/6-test-group-stealing/foo/test/src/foo/GroupX2.java
package foo;

import org.junit.Test;

public class GroupX2 extends RandomTestsUtils {
  @Test
  public void test1() throws Exception {
    testGreeting("Chronos", 35);
  }

  @Test
  public void test2() throws Exception {
    testGreeting("Demeter", 31);
  }

  @Test
  public void test3() throws Exception {
    testGreeting("Eos", 32);
  }
}

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupX2.kt
package foo

import org.junit.jupiter.api.Test

class GroupX2 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() { testGreeting("Chronos", 35) }
    
    @Test
    @Throws(Exception::class)
    fun test2() { testGreeting("Demeter", 31) }
    
    @Test
    @Throws(Exception::class)
    fun test3() { testGreeting("Eos", 32) }
} 